subworkflows/local/gens.nf

```diff
@@ -36,8 +36,8 @@ workflow GENS {
 
         COLLECTREADCOUNTS.out.hdf5
             .branch { meta, counts ->
-                female: meta.sex.equals(2) || meta.sex.equals(0)
-                male: meta.sex.equals(1)
+                female: meta.sex.equals('2') || meta.sex.equals('other')
```

Comment:
Looks fine to me, but have you checked a test case through this to make sure that this works as expected? :) 

And perhaps you can use matches instead of equals here?

Reply:
I tried enabling the gens subworkflow and ran a stub run and was then able to get collectreadscounts to execute, which it didn't before. 

Good point about matches. Will update